src/components/Typeahead/Typeahead.less

```diff
@@ -100,6 +89,13 @@
       background-color: var(--gray-10);
     }
   }
+
+  .o-search-input__input {
+    //display: block;
+    &-label {
+      z-index: 1;
```

Comment:
Why's this necessary? Maybe add a code comment. Is it possible to put the z-index on the parent `o-search-input`, so that you're not customizing the component internals?

Reply:
without the z-index, the spyglass is appearing under the input box.  

I think this is due to how the typeahead works.
With the code it looks like:
<img width="353" alt="Screenshot 2024-06-14 at 11 13 43 AM" src="https://github.com/cfpb/ccdb5-ui/assets/1038306/5fd7c9fc-60ef-44d9-ad9c-193c18cd9346">
without this code:
<img width="359" alt="Screenshot 2024-06-14 at 11 14 00 AM" src="https://github.com/cfpb/ccdb5-ui/assets/1038306/3d64693e-c8c5-43bd-8cf2-54712c2f95ec">